src/components/Navbar.js

```diff
@@ -0,0 +1,11 @@
+const Navbar = () => (
+  <nav className="navbar">
+    <h1> BookStore CMS </h1>
+    <div className="links">
+      <a href="/">Books</a>
+      <a href="/Category">Categories</a>
```

Comment:
- [ ] I think it would be better if you import `Link ` from react-router-dom and then use it to link the books and categories 👍🏼 .

e.g. 

import {Link} from 'react-router-dom'

      <Link href="/">Books</Link>
      <Link href="/Category">Categories</Link>